packtools/sps/validation/article_authors.py

```diff
@@ -318,6 +319,79 @@ def validate_authors_orcid_is_registered(self, callable_get_validate=None):
                 'advice': None if is_valid else "ORCID {} is not registered to any authors".format(orcid)
             }
 
+    def validate_authors_collab_list(self):
+        """
+        Checks if there is identification of authors for a group of collaborators.
+
+        XML input
+        ---------
+        <article>
+            <front>
+                <article-meta>
+                    <contrib-group>
+                        <contrib contrib-type="author">
+                            <collab>The MARS Group</collab>
+                        </contrib>
+                    </contrib-group>
+                    <contrib-group content-type="collab-list">
+                        <contrib contrib-type="author" rid="collab">
+                        <contrib-id contrib-id-type="orcid">0000-0001-0002-0003</contrib-id>
+                        <name>
+                        <surname>Wright</surname>
+                        <given-names>Rick W.</given-names>
+                        </name>
+                        </contrib>
+                    </contrib-group>
+                </article-meta>
+            </front>
+        </article>
+
+
+        Returns
+        -------
+        list of dict
+           A list of dictionaries, such as:
+            [
+                {
+                    'title': 'Collab list authors identification',
+                    'parent': None,
+                    'parent_id': None,
+                    'item': 'contrib-group',
+                    'sub_item': '@content-type',
+                    'validation_type': 'exist',
+                    'response': 'ERROR',
+                    'expected_value': 'contrib group with identification of members of The MARS Group',
+                    'got_value': None,
+                    'message': 'Got None, expected contrib group with identification of members of The MARS Group',
+                    'advice': 'provide the identification of members of The MARS Group',
+                    'data': {
+                        'aff_rids': None,
+                        'collab': 'The MARS Group',
+                        'contrib-type': 'author'
+                    }
+                }...
+            ]
+        """
+        content_types = []
+        for contrib_group in self._xmltree.xpath('.//contrib-group'):
+            content_types.append(contrib_group.get('content-type'))
+        for contrib in self.article_authors.all_contribs:
+            collab = contrib.contrib_with_aff.get('collab')
```

Comment:
@Rossi-Luciano Trabalhe com 3 classes:
- ArticleContribs
- Contribs (contextualizado no article, contextualizado no sub-article)
- Contrib

Em Contrib, deve conter todas as validações relacionadas com o contrib.

Sendo assim, não deve haver um método que fará um FOR em todos os contribs para validar o contrib-list.

Em outras palavras, é desejável iterar por cada contrib e validar todos os componentes de um contrib. Não é desejável, ter uma validação de cada compontente e criar uma sequência de validações como está nas linhas 401-404